sdk/storage/azdatalake/internal/generated_blob/autorest.md

```diff
@@ -347,11 +347,13 @@ directive:
 
 ``` yaml
 directive:
-  - from: zz_service_client.go
-    where: $
-    transform: >-
-      return $.
-        replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`)
+- from: 
+  - zz_service_client.go
+  - zz_container_client.go
+  where: $
+  transform: >-
+    return $.
+      replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`);
```

Comment:
```suggestion
      replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/g, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`);
```

Reply:
changed

Reply:
The `/g` parameter is not added yet.
https://github.com/Azure/azure-sdk-for-go/blob/98591cf681ee73baadf93884f201d3ffbefb13d0/sdk/storage/azdatalake/internal/generated_blob/zz_container_client.go#L162
https://github.com/Azure/azure-sdk-for-go/blob/98591cf681ee73baadf93884f201d3ffbefb13d0/sdk/storage/azdatalake/internal/generated_blob/zz_container_client.go#L162

Reply:
It is added now